packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -51,7 +50,19 @@ function useDebouncedValue( value ) {
 const getNavigationCommandLoaderPerPostType = ( postType ) =>
 	function useNavigationCommandLoader( { search } ) {
 		const history = useHistory();
-		const isBlockBasedTheme = useIsBlockBasedTheme();
+		const { isBlockBasedTheme, canCreateTemplate } = useSelect(
+			( select ) => {
+				return {
+					isBlockBasedTheme:
+						select( coreStore ).getCurrentTheme()?.is_block_theme,
+					canCreateTemplate: select( coreStore ).canUser( 'create', {
+						kind: 'postType',
+						name: 'wp_template',
```

Comment:
Should we be checking "postType" here rather than `wp_template`

Reply:
If we change this to `postType`, it won't work as intended. For example, non-admin users will be able to create Pages/Posts but won't have access to the site editor.

Reply:
Maybe there's something I'm missing. Oh so the links go to the site editor? Maybe the links should go to the post editor in that case (if you're not already in the site editor)?